ansible/setup.yml

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
I think we ended up not using this file, we can simplify the codebase and remove it.